themes/Metro.theme.xml

```diff
@@ -594,20 +594,15 @@
                                 <VisualState x:Name="Unfocused"/>
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
-                        <TextBlock x:Name="UnderlineTextBlock"
-                                   HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-                                   Margin="{TemplateBinding Padding}" Text="{TemplateBinding Content}"
-                                   TextDecorations="Underline" Visibility="Collapsed"
-                                   VerticalAlignment="{TemplateBinding VerticalContentAlignment}">
-                            <TextBlock.Foreground>
+                       
+                        <Border x:Name="UnderlineTextBlock" Margin="2,0,2,1" BorderThickness="0,0,0,1" Visibility="Collapsed">

```

Comment:
The more I think about it...we should just apply `TextDecorations = Underline` should be applied to the content presenter on hover.  Since TextDecorations is inheritable, it should cascade to the TextBlock in the template regardless of how it was created.